gradle.properties

```diff
@@ -1 +1 @@
-systemProp.version = 2.1.1
\ No newline at end of file
+systemProp.version = 3.0.0-SNAPSHOT
```

Comment:
@reta We should remove the "-SNAPSHOT" from the system property and opensearch versions since this is already appended when published to sonatype. 

Currently, 3.0.0 snapshots are being published incorrectly :

![image](https://user-images.githubusercontent.com/99766446/199767316-86d98dcf-abe8-4715-9146-db46fcfc7fa2.png)

CC : @VachaShah

Reply:
@joshpalis check please https://github.com/opensearch-project/opensearch-java/pull/254